tests/functional/Loan.ts

```diff
@@ -0,0 +1,37 @@
+
+import FineractAPI, { FineractAPIConfig } from "../../FineractAPI";
+import { before, after } from "intern/lib/interfaces/bdd";
+const { describe, it } = intern.getInterface('bdd');
+const { assert } = intern.getPlugin('chai');
+const test_path ="test_loans";
+const test_loan_config={random:"value"};
+const test_response_data={ "totalFliteredRecords":1,"pageItems":[{"id": 1,"accountNo": "000000001",}]};
+
+let config: FineractAPIConfig;
+describe('Loan Tests', () => {
+
+    before(() => {
+        require('dotenv').config();
+
+        config = {
+            client_base_url: process.env.CBS_BASE_URL as string,
+            client_username: process.env.CBS_USER as string,
+            client_password: process.env.CBS_PASSWORD as string,
+            client_tenant_id: process.env.CBS_TENANT as string
+        };    
+    
+    });
+
+    it(`Loan :Lists loan success message`, () => {
+        return (async () => {
+            let response = await (new FineractAPI(config)).get('list_loan', { path: test_path, query:test_loan_config });
```

Comment:
we should be using the newly created `Loan` class here and calling `list_loan` on it.